docs/source/index.rst

```diff
@@ -0,0 +1,36 @@
+.. MatchZoo documentation master file, created by
+   sphinx-quickstart on Mon May 28 16:40:41 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to MatchZoo's documentation!
+====================================
+
+
+.. image:: https://travis-ci.org/faneshion/MatchZoo.svg?branch=master
+   :alt: ci
+   :target: https://travis-ci.org/faneshion/MatchZoo/
```

Comment:
The URL should be `https://travis-ci.org/NTMC-Community/MatchZoo-py.svg?branch=master`